src/timeMachine/components/RawFluxDataTable.tsx

```diff
@@ -23,11 +27,15 @@ class RawFluxDataTable extends PureComponent<Props, State> {
   public state = {scrollLeft: 0, scrollTop: 0}
 
   private parseFiles = memoizeOne(parseFiles)
+  private parseFilesWithObjects = memoizeOne(parseFilesWithObjects)
 
   public render() {
     const {width, height, files, disableVerticalScrolling} = this.props
+
     const {scrollTop, scrollLeft} = this.state
-    const {data, maxColumnCount} = this.parseFiles(files)
+    const {data, maxColumnCount} = isFlagEnabled('parseObjectsInCSV')
```

Comment:
dont forget to update config cat.. THEN update idpe to get this flag into production